ow_plexil/src/plexil-adapter/OwAdapter.cpp

```diff
@@ -435,7 +435,7 @@ bool OwAdapter::initialize()
   g_configuration->registerCommandHandler("arm_move_joints", arm_move_joints);
   g_configuration->registerCommandHandler("dig_circular", dig_circular);
   g_configuration->registerCommandHandler("dig_linear", dig_linear);
-  g_configuration->registerCommandHandler("deliver", deliver);
+  g_configuration->registerCommandHandler("taskDeliverSample", taskDeliverSample);
```

Comment:
Suggest not making this change, because it will require a ton of PLEXIL plan editing.  I don't think the PLEXIL command names need to match the ROS action names.